src/auro-toast.js

```diff
@@ -32,7 +33,11 @@ export class AuroToast extends LitElement {
   // to understand how to use reflected attributes with your property settings.
   static get properties() {
     return {
-      // ...super.properties,
+      ...super.properties,
+      type: {
+        type: String,
+        reflect: true
+      }
```

Comment:
LOL i thought you meant this should be a variant property and i was like ummm what is that... and couldn't find anything about the in the LitElement documentation. but now i realize you're talking about the name of this property should be `variant` instead of `type` 😆 (am thinking correctly?)

anyway... the reason it's not being used in the HTML template is because this variant only effects the the color of the element (and icon which is the issue to be worked on next)